### PR TITLE
Minor changes to pass week-7 checks

### PR DESCRIPTION
**Pull request by cindyangXQ**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### HsiaoJietNg
 
-<img src="images/team/hsiaojietng.jpeg" width="200px">
+<img src="images/hsiaojietng.jpeg" width="200px">
```
Comment (Ashuh) [inline docs/AboutUs.md:44]: File extension should be `.png` as described [here](https://nus-cs2103-ay2122s2.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme).

**Review by Ashuh — CHANGES_REQUESTED** [review]
Looks good but Jiet's profile picture extension needs to be changed.

**Comment by waffledood** [comment]
I've converted Jiet's picture to `.png` format, available here, you can right click & choose "Download" 
hsiaojietng

**Review by waffledood — APPROVED** [review]
LGTM! 👍

**Review by Ashuh — APPROVED** [review]
LGTM

**Pull request merged**